src/core_plugins/timelion/server/fit_functions/average.js

```diff
@@ -27,7 +27,7 @@ module.exports = function (dataTuples, targetTuples) {
 
     dataTuplesQueue.splice(0, i);
 
-    const sum = _.reduce(avgSet, function (sum, num) { return sum + num; }, 0);
+    const sum = avgSet.reduce((sum, num) => sum + num);
```

Comment:
Thanks @thomasneirynck for the review! I made the changes and it should work now.